### ISSUE TITLE
haskell.org needs a governance page

### ISSUE THREAD

**Issue by ketzacoatl** [haskell-infra/www.haskell.org#60]
The haskell community is large and made up of many groups. These groups have different, and sometimes overlapping responsibilities. To a new haskeller, it makes no sense and there are no clear answers on the confusion. Desires to contribute and help in small ways take us down rabbitholes looking for which group is responsible for what and who to ask for permission or who to get approval for a change. This leads to contributors giving up before even sending in their contribution, and creates barriers to entry.

Given the complicated nature of the haskell ecosystem and the various groups with decision making power, when I look at haskell.org, I expect to see a page that _clearly and concisely_ lists those groups and the overall governance structure for the community.

For comparison, https://www.rust-lang.org/governance is both clear and concise (they also have some of the same structures found in the haskell ecosystem).

I'd be happy to work with the haskell.org site developers to help craft such a page provided that I receive:
* confirmation a new page with this content would be welcome and accepted
* agreement the page should be included in the main list of links on the haskell.org website
* support and feedback listing the groups and determining the content for the page

**Comment by ketzacoatl** [comment]
@TikhonJelvis / @emilypi, maybe you have feedback on this topic? What is the general consensus about this, and what would that translate to for next steps?

**Comment by emilypi** [comment]
@ketzacoatl This is a good ask. Historically (and currently), our governance page has existed as an entry in the HaskellWiki: https://wiki.haskell.org/Haskell.org_committee. However, I do think it would be useful to create a page for this information on the org site. I leave it to the committee to decide this, though.

**Comment by ketzacoatl** [comment]
Hello @emilypi, I should clarify.. there is already a page for that content you reference - https://www.haskell.org/haskell-org-committee/. Also, the haskell.org committee would be just one entity explained on the governance page - I came to haskell.org looking for this type of information (who is responsible for X within the haskell ecosystem), and the committee page did not answer those questions.

**Comment by emilypi** [comment]
Feel free to write up a POC for what that would look like. I'm sure the committee would be happy to review

**Comment by TikhonJelvis** [comment]
This sounds pretty close to what the Haskell Foundation is planning to organized with it's "associated organizations"—which includes Haskell.org along with several other groups and committees. Once the Foundation and these efforts are in a slightly more mature state, I imagine it will make sense to translate some of that content to the haskell.org website itself.

**Comment by ketzacoatl** [comment]
Yes, that would be great. Though I guess we should plan to do our own version if there isn't an alternative available within 3 months?

**Comment by ketzacoatl** [comment]
@TikhonJelvis should we start on an effort to define the elements on this page or description of the orgs to be included, or are we blocked on a particular output from the Foundation?

**Comment by hasufell** [comment]
> and the overall governance structure for the community.

I'm not sure we have that, do we?

I believe haskell.org should just list known bodies and how it interacts with/supports them (CLC, GHC dev team, HF, ...).

Contribution processes should be documented by the respective bodies.

This might be a bit vague, but that accurately reflects the current nature of the situation.

**Comment by ketzacoatl** [comment]
Is there any interest in defining and implementing a solution that addresses the issue?

**Comment by ketzacoatl** [comment]
Sure, though I would push for the page being on the haskell.org website, as that is the de-facto landing/front-page for Haskell newbs.

**Comment by tomjaguarpaw** [comment]
Seems reasonable for haskell.org to host it but I think that HF should prioritise it amongst the other things they have on their plate and take responsibility for getting it done.

**Comment by ketzacoatl** [comment]
cc @emilypi / @myShoggoth

**Comment by myShoggoth** [comment]
I agree with this notion in principle.

**Comment by jaspervdj** [comment]
While the HF may have some more bandwidth to tackle this, there is maybe some institutional knowledge as well; so I'm happy to start a draft this weekend and then we can go from there.

**Comment by ketzacoatl** [comment]
Has there been any traction or progress towards drafting content for this governance page?

**Comment by silky** [comment]
Would be great to get some progress here!
